modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/js/miller_columns/KeyboardMovementText.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import React, {useEffect} from 'react';
+
+import {
+	useMovementText,
+	useSetMovementText,
+} from './contexts/KeyboardMovementContext';
+
+export default function KeyboardMovementText() {
```

Comment:
Since this component is also used in the page editor, can we move it to `layout-components-js` and use it from there?

Reply:
Yes it makes sense